libsolidity/ast/Types.cpp

```diff
@@ -414,6 +414,14 @@ u256 IntegerType::literalValue(Literal const* _literal) const
 	return u256(_literal->value());
 }
 
+s256 IntegerType::signedValue(Literal const* _literal) const
```

Comment:
This doesn't seem to be used yet?

I wonder if `rational value()` would be a better choice since it would be a unified interface between this and `RationalNumberType`?